tests/e2e/e2e.go

```diff
@@ -584,6 +587,37 @@ func (ctx *TestCtx) GetDerivedValueStateClb(baseValue proto.Message, derivedKey
 	}
 }
 
+// GetValueMetadata retrieves metadata associated with the given value.
+func (ctx *TestCtx) GetValueMetadata(value proto.Message, view kvs.View) (metadata interface{}) {
+	type KeyWithMetadata struct {
+		Key      string
+		Metadata interface{}
+	}
+	key, err := models.GetKey(value)
+	if err != nil {
+		ctx.t.Fatalf("Failed to get key for value %v: %v", value, err)
+	}
+	model, err := models.GetModelFor(value)
+	if err != nil {
+		ctx.t.Fatalf("Failed to get model for value %v: %v", value, err)
+	}
+	q := fmt.Sprintf(`/scheduler/dump?key-prefix=%s&view=%v`, url.QueryEscape(model.KeyPrefix()), view)
+	resp, err := ctx.httpClient.GET(q)
+	if err != nil {
+		ctx.t.Fatalf("Request to dump values failed: %v", err)
+	}
+	var kmDump []KeyWithMetadata
+	if err := json.Unmarshal(resp, &kmDump); err != nil {
+		ctx.t.Fatalf("Reply with dumped values cannot be decoded: %v", err)
+	}
```

Comment:
Perhaps we could reuse the client for agentctl which provides this API for KVscheduler:

```suggestion
c, err := client.NewClient(client.DefaultAgentHost)
if err != nil {
	ctx.t.Fatal(err)
}
kmDump, err := c.SchedulerDump(context.Background(), types.SchedulerDumpOptions{
	KeyPrefix: model.KeyPrefix(),
	View:      view.String(),
})
```

and possibly further simplify it by instantiating the client during the Setup.

Reply:
I have updated `e2e.go` to use client from agentctl for all interactions with the agent - please see the second commit.
I expect that there will be some merge conflicts with https://github.com/ligato/vpp-agent/pull/1770, but they should be relatively easy to fix.